tests/unit/mixin/form-field-options-test.js

```diff
@@ -0,0 +1,32 @@
+import { moduleFor, test } from 'ember-qunit';
+
+import FormFieldOptionsMixin from
+  'ember-railio-form-components/mixins/form-field-options';
+
+import Component from 'ember-component';
+import getOwner  from 'ember-owner/get';
+import get       from 'ember-metal/get';
+
+moduleFor('mixin:form-field-options', 'Unit | Mixin | form-field-options', {
+  integration: true,
+
+  subject() {
+    let formComponent = Component.extend(FormFieldOptionsMixin, {
+      options: {
+        fooBar: 'Value for FooBar',
+        fizBox: () => 'Value for FizBoz'
+      }
+    });
+
+    this.register('component:form-field', formComponent);
+    return getOwner(this).lookup('component:form-field');
```

Comment:
Do you have to register it, or will it work on returning the `formComponent` directly?

Reply:
Didn't work